riocli/v2client/client.py

```diff
@@ -120,6 +119,11 @@ def create_project(self, spec: dict) -> Munch:
         """
         url = "{}/v2/projects/".format(self._host)
         headers = self._get_auth_header(with_project=False)
+
+        # Set the organizationguid header
+        if spec['metadata'].get('organizationGUID'):
+            headers['organizationguid'] = spec['metadata'].get('organizationGUID')
```

Comment:
Do we need this ?

Reply:
I feel that we should still support this since a manifest may have the organizationGUID defined explicitly in it. It effectively goes via the header, so we need not worry about the backend handling it in any other way.